engines/dfc_provider/app/controllers/dfc_provider/api/catalog_items_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# Controller used to provide the API products for the DFC application
+module DfcProvider
+  module Api
+    class CatalogItemsController < DfcProvider::Api::BaseController
+      def index
+        render json: current_user, serializer: DfcProvider::PersonSerializer
```

Comment:
In the corresponding spec for the `CatalogItemsController#index` action, the test is checking if a variant is rendered, but in the controller action here it's rendering a user..?

Reply:
Shouldn't this action be a list of variants?

Reply:
I agree it is not super clear but the DFC specification is nesting the items catalog into an enterprise, and then the entreprise into an user. I will check out if it is designed by choice or by necessity (semantic specification).

Reply:
:-O if you need to keep this as is, it needs a comment explaning it, ok?

Reply:
Added a comment here, https://github.com/openfoodfoundation/openfoodnetwork/pull/6158/commits/2de150f11fa0e61f23daaf0a6b558804749c217c, is it ok?

Reply:
that's acceptable yes :+1: 
But from an API design perspective, I think this is wrong. This Person object with affiliates inside (and items inside) should be on the /persons endpoint, for me this would be the index action on the PersonsController, not the CatalogItems endpoint.